lib/network/rpc/utils.js

```diff
@@ -40,7 +40,9 @@ module.exports.reconstructError = function reconstructError(err) {
         return err;
     }
     const reconstructedErr = new Error(err.message);
-
+    reconstructedErr.is = {
```

Comment:
@jbertran I added this so we can use `is` when dealing with these errors, it feels kinda awkward as they are not really arsenal errors but it also felt weird to not have `.is` in the bucket file interface.